src/openapi.json

```diff
@@ -484,27 +488,27 @@
         }
       },
       "post": {
-        "summary": "Add a new application. (In Development)",
+        "summary": "Add a new proposal. (In Development)",
```

Comment:
I think the "version = 0.1.0" and "pilot" etc. all communicate the idea that this whole API is in development, so I would leave out this comment. At the same time, when exploring the API using the OpenAPI/swagger interface, it is nice to know which parts are expected to work and not. Then again, if it's not going to work, we probably shouldn't have it in the `openapi.json` yet, right?

Reply:
In any case this comment does not block merging in any way. Just a thought.

Reply:
Ah -- this was in there for the demo since I wanted to help clarify which API endpoints were working and which were just documentation placeholders.  I'd like to keep it insofar as folks might be looking at the swagger spec and wanting to be able to predict which endpoints they can trust actually exist.  It's about to be taken out though since it's almost done!